Update sandbox client packages install script

sharifsalah opened the pull request:
**Link the Issue(s) this Pull Request is related to.**

Fixes #467 

**Summarize your change.**

Adds a script to install client packages for a specific version of OpenCue to run against the sandbox environment. After review and approval, I'll update the quick starts on opencue.io before I submit this change to make sure the docs and code are in sync. I'll update the quick starts to explain that users can try installing from the release archives, but can install from source as an alternative.

bcipriano commented on sandbox/install-client-archives.sh:
```diff
@@ -0,0 +1,33 @@
+# For more information, see README.md
+
+if [[ -z "${VERSION}" ]]; then
+    echo "You must set the release version number. For example:"
```
Should we add the URL to the release pages here, to make it easy for folks to look up which version number they're interested in?

sharifsalah replied:
Good idea, thanks. I'll add the URL.

bcipriano commented on sandbox/install-client-archives.sh:
```diff
@@ -0,0 +1,33 @@
+cd opencue-downloads
+
+for PACKAGE in "${CLIENT_PACKAGES[@]}"; do
+    wget ${BASE_URL}${VERSION}/${PACKAGE}-${VERSION}-all.tar.gz
```
Is `wget` included with macOS by default? I seem to remember that it's not, and while I don't have a fresh mac on hand to test this out with, my workstation seems to only have a homebrew-installed one:

```
$ which -a wget
/Users/cipriano/homebrew/bin/wget
```

Should we add it to the mac quickstart?

sharifsalah replied:
I don't believe it's included by default. I don't have it on any of my machines.

Yes, agree about adding it to the mac quick start. I plan to update the quick starts over the next couple of days so that you can review the two PRs side-by-side. I wanted to get your initial reaction to this approach before updating the docs. Thanks for the quick review and the enthusiastic response.

sharifsalah replied:
Done.

sharifsalah commented:
Thanks again for the quick review. I've updated the script with some additional changes:

- Added a URL for the release version numbers.
- Added a secondary URL format for `wget` to try. It looks like this recently changed and am assuming we're sticking with the latest format?
- Swapped the order the packages get installed.
- Added a URL to point at the quick starts on opencue.io

sharifsalah commented on sandbox/install-client-archives.sh:
```diff
@@ -0,0 +1,40 @@
+    echo "You must set the release version number. For example:"
+    echo "export VERSION=0.2.31"
+    echo "For a list of OpenCue version numbers, visit the following URL:"
+    echo "https://github.com/AcademySoftwareFoundation/OpenCue/releases/" 1>&2
```
Thanks for checking. I remember I meant to include this on all lines and forgot. Looking at the other scripts under `/ci`, this format isn't used in OpenCue. So for consistency I'll remove it.
